### PR TITLE
Novacustom ns5x/rgb keyboard

### PR DESCRIPTION
**Pull request by mkopec**
TODO: Call `KBUP ()` in a more appropriate place

### CODE REVIEW

File: src/ec/clevo/it5570/acpi/rgb_kb.asl [hunk 1]
```diff
@@ -0,0 +1,105 @@
+Name (KBEN, 0x01) // Keyboard Backlight Enable
```
Comment (miczyg1) [inline src/ec/clevo/it5570/acpi/rgb_kb.asl:1]: license header missing. It would be good to issue `make test-lint` when issuing a PR

**Review by miczyg1 — CHANGES_REQUESTED** [review]
Looks, good, just need the license header in the added file

**Comment by miczyg1** [comment]
@mkopec one concern: doesn't it conflict with the single color backlight on NV41? We should think about two platforms when developing anything in the ec directory.

**Comment by mkopec** [comment]
@miczyg1 I tested this on an nv41, everything regarding white keyboard backlight still works as it should. No side effects after closing the lid or sleeping.

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**